doc/admin-guide/configuration/hrw4u.en.rst

```diff
@@ -220,6 +225,13 @@ set-debug         set-debug()                  Enables ATS txn debug
 skip-remap        skip-remap()                 Skip remap processing (open proxy)
 ================= ============================ ================================
 
+String concatenations
+---------------------
+
+You can concatenate values using strings, condition values and variable expansions on the same line in
+operators using. For instance, `outbound.req.CustomHeader “Hello from {inbound.ip}:{inbound.port}”`. As
+a result, the set-redirect's `[QSA]` flag would be implemented as `set-redirect(302, "https://...{inbound.url.query}")``
```

Comment:
Maybe make sure the ? is here? I'm not sure if ATS's query value includes the ? or not... I want to say not but I can test tomorrow.

Reply:
Updated.